deskStructure.js

```diff
@@ -121,13 +135,41 @@ export default () => {
                 S.documentTypeList('ecologicalOutcome')
                 .title('Ecological Outcomes')
               ),
+            S.listItem()
+              .title('Land Management Practices')
+              .schemaType('landManagementPractice')
+              .child(
+                S.documentTypeList('landManagementPractice')
+                .title('Land Management Practices')
+              ),
             S.listItem()
               .title('SDGs')
               .schemaType('sdg')
               .child(
                 S.documentTypeList('sdg')
                 .title('SDGs')
               ),
+            S.listItem()
+              .title('Timeline Items')
```

Comment:
I'm not sure if that makes sense to have that as part of the document types, is this kind of timeline items used elsewhere? Do we wanna be able to use the same timeline item in different places? If not, I'd suggest to make it just a regular sanity `object`

Reply:
aggreed. object now